charts/rasa-x/values.yaml

```
@@ -194,6 +194,54 @@ rasa:
       # - name: tmpdir
       #   emptyDir: {}
 
+
+# dbMigrationService specifies settings for the database migration service
+# The database migration service requires Rasa X >= 0.33.0
+dbMigrationService:
+  # settings related to the init-db container used as a init container for services
+  initContainer:
+    image: alpine:latest
+  # command overrides the default command to run in the container
+  command: []
+  # args overrides the default arguments to run in the container
+  args: []
+  # name is a Docker image name
```

optional: Just to make sure users are not confused between the init container image and this one
```suggestion
  # name is the Docker image name which is used by the migration service
```